content/hi/docs/contribute/localisation_hi.md

```diff
@@ -0,0 +1,71 @@
+---
+title: कुबेरनेटिस प्रलेखन के स्थानीयकरण के लिए एक गाइड
+content_type: concept
+---
+<!-- overview -->
+
+कुबेरनेटिस प्रलेखन के स्थानीयकरण के लिए एक गाइड
+
+<!-- body -->
+
+## टीम लक्ष्य प्रबंधन
+
+कुबेरनेटिस दस्तावेज़ स्थानीयकरण टीम [समुदाय की स्थानीयकरण मार्गदर्शिका](/docs/contribute/localization/#branching-strategy) के अनुसार स्थानीयकरण के लिए टीम के लक्ष्य और विकास शाखाओं का प्रबंधन करती है। यह खंड हिंदी भाषा टीम के टीम लक्ष्य प्रबंधन के लिए विशिष्ट सामग्री से संबंधित है।
+
+हिंदी भाषा टीम `main` शाखा से शाखित शाखा का उपयोग करती है। और विकास शाखा के नाम में निम्नलिखित संरचना है:
+`dev-1.22-hi.1`
+आम तौर पर, नवीनतम रिलीज शाखा से अपनी कार्यशील शाखा बनाएं। नवीनतम रिलीज़ शाखा का नाम समय के साथ बदलता है। उदाहरण के लिए, जब हम `dev-1.22-hi.2` को `main` में मिलाते हैं, तो नई `/hi` स्थानीयकरण शाखा का नाम `dev-1.22-hi.2` हो सकता है। और उदाहरण के लिए, जब हम अंततः Kubernetes `1.25` दस्तावेज़ीकरण का स्थानीयकरण शुरू करते हैं, तो नई `/hi` स्थानीयकरण शाखा का नाम `dev-1.25-hi.1` हो सकता है।
+विकास शाखा का उपयोग लगभग 2 से 3 सप्ताह की टीम लक्ष्य अवधि के दौरान सहयोगात्मक कार्य के लिए किया जाता है और टीम लक्ष्य समाप्त होने पर मूल शाखा में विलय कर दिया जाता है।
+
+## शैली गाइड
+
+### सम्मान
+सिद्धांत रूप में, सादा भाषा का उपयोग किया जाता है, लेकिन केवल कुछ पृष्ठों के लिए अपवाद हैं (जैसे, [https://kubernetes.io/en](https://kubernetes.io/en))।
+उच्च स्तर के शब्दों का प्रयोग करें।
+
+### अनिवार्य 
+अनिवार्य अभिव्यक्ति का एक सरल भाषा में अनुवाद किया जाता है जो श्रोता को कुछ करने के लिए कहता है,जिसका उद्देश्य संक्षिप्त और कोमल अभिव्यक्ति में सटीक अर्थ व्यक्त करने के लिए एक साथ कुछ करना है।
+
+आदेश सूचक वाक्य | अनुवाद
+--- | ---
+साइट पर जाएँ | आइए साइट पर जाएँ 
+गाइड देखें | गाइड का संदर्भ लें (साधारण भाषा में)
+
+### इन प्रकार के शब्दों और वाक्यांशों से बचें
+
+अनावश्यक रूप से औपचारिक स्वर से बचा जाता है। यहाँ कुछ उदाहरण दिए गए हैं
+शब्दों और वाक्यांशों से बचने के लिए।
+
+en-US स्रोत | hi-IN बचने के लिए शब्द | hi-IN शब्द/वाक्यांश
+--- | --- | ---
+Select/choose/pick | | 
+Furthermore | इसकेअनतररक्त | साथही
+Therefore | अत: | इसीलिए
+Refer to | सन्दर्भ ले | देखें
+Change | परिवर्तित करे | बदले
+Deselect | इसका चयन रद्द करे | अचयनित करे
+Pump | पम्प | पंप
+
+
+### सामान्य संक्षिप्ताक्षर
+
+स्थान की कमी के कारण आपको UI (मुख्य रूप से बटन और विकल्पों के नाम) में कुछ शब्दों को संक्षिप्त करने की आवश्यकता हो सकती है। हिंदी में, शब्दों के लिए कोई मानकीकृत संक्षिप्ताक्षर नहीं हैं, इसलिए शब्द के सबसे छोटे उपलब्ध संस्करण का उपयोग करने का प्रयास करें। इसके अलावा, मेनू में, कोशिश करें
```

Comment:
I'm sure the second sentence is incomplete. Please can you recheck?
```suggestion
स्थान की कमी के कारण आपको UI (मुख्य रूप से बटन और विकल्पों के नाम) में कुछ शब्दों को संक्षिप्त करने की आवश्यकता हो सकती है। हिंदी में, शब्दों के लिए कोई मानकीकृत संक्षिप्ताक्षर नहीं हैं, इसलिए शब्द के सबसे छोटे उपलब्ध संस्करण का उपयोग करने का प्रयास करें। इसके अलावा, मेनू में, कोशिश करें
```

Reply:
fixed with https://github.com/kubernetes/website/pull/36133/commits/c4b861a30b90c716553bdb3f4cb0d5f0a1629c43